specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/blob.json

```diff
@@ -1142,6 +1142,10 @@
             "changeFeed": {
               "$ref": "#/definitions/ChangeFeed",
               "description": "The blob service properties for change feed events."
+            },
+            "containerDeleteRetentionPolicy": {
```

Comment:
the neww property will be think as breaking change in stable version. suggest to add it in new version

Reply:
@yungezz 
I don't see any reason why add new properties are breaking change, would you please give some reason. 
And we have add new properties in same API verison before, but not take as breaking.
Per my understanding, REmove/modify a property will breaking, but add new properties are not breaking.

Reply:
https://github.com/Azure/adx-documentation-pr/wiki/Breaking-changes-guidelines#new-required-property-added-to-request, 
https://github.com/Azure/adx-documentation-pr/wiki/Breaking-changes-guidelines#new-property-added-to-response

Reply:
Yunge, I understand your concern. Adding a new property can cause data lost during update. June19 is the version we added last week. We can't afford to add a new version for every new feature. As Wei mentioned, we have been adding properties in the same API version before, which does not cause any issue.

Reply:
@yungezz
Thanks for the sharing, from the doc I see : "This will overwrite the change made by the first customer from the portal."

Per my understanding, in current storage object update request, if the request send to server does not contain a property, the property keep the value and won't be updated. @zfchen95 , would you please confirm?
In this case, the GET-PUT pipeline won't be broken, so there are no breaking change.

Reply:
Correct. If the property is not shown in the request, it will not clear the property.

Reply:
@yungezz 
Do you agree adding the new property in this PR is not a breaking? since it won't break the GET-PUT pipeline. Let us know if you still have concern on this.

Reply:
@blueww as long as service team can make sure no user breaking experience, we're ok to it.